### DISCUSSION TITLE
eslint throwing "Parsing error: This experimental syntax requires enabling one of the following parser plugin"

### DISCUSSION THREAD

**Discussion by ljmerza** [tailwindlabs/tailwindcss#8560] [Help]
I have a basic react setup with Vite, tailwind, and eslint. My tailwind config is:

```
{
  "extends": ["react-app", "plugin:tailwindcss/recommended"],
  "rules": {
    "import/no-default-export": "error",
    "tailwindcss/classnames-order": "warn",
    "tailwindcss/enforces-negative-arbitrary-values": "warn",
    "tailwindcss/enforces-shorthand": "warn",
    "tailwindcss/migration-from-tailwind-2": "warn",
    "tailwindcss/no-arbitrary-value": "off",
    "tailwindcss/no-custom-classname": "warn",
    "tailwindcss/no-contradicting-classname": "error"
  },
  "plugins": ["tailwindcss"]
}

```

my eslint config is:

```
module.exports = {
  darkMode: 'class',
  content: ['./src/**/*.{js,jsx,ts,tsx}'],
  theme: {
    extend: {},
  },
};

```

then in my `index.css` file i import tailwind:

```
@tailwind base;
@tailwind components;
@tailwind utilities;

```

when i try to launch the app though i get `1:0  error  Parsing error: This experimental syntax requires enabling one of the following parser plugin(s): "decorators-legacy", "decorators". (1:0)`. Eslint is not picking up the tailwind imports and the eslint "decorators-legacy" library is depricated. How do i get eslint working?

**Comment by ywong-cais** [comment]
seeing the same problem, anyone know a solution?

**Comment by wongjn** [comment]
I'm assuming you've got the ESLint and Tailwind configs mixed up in your post. Try removing each of the configs in ESLint `extends` and the ESLint `tailwindcss` plugin. Does the error go away when one of these is removed? If so, then it could be to do with that plugin/package.

**Reply by Florian-crg** [reply]
Unfortunately this ins't linked to the rules.  On my side I had this error even before adding the rules and tailwind plugin.
